### PR TITLE
Add 4JNET (BEP20)

### PR DESCRIPTION
**Pull request by bronejoost**
Adding info for BEP20 token '4JNET'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by bronejoost** [comment]
logo

not sure what I have to do more than this, sorry (doing my utmost best)

**Comment by bronejoost** [comment]
I also like to add the NFT address, should I add it separate
0xc942CEee2D86E3e066Ac1f97044E911ced2fe314

File: blockchains/smartchain/assets/0xbfb1a68962Fb4ED040FD3a0a71dC2C2015BCc667/info.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            "name": "twitter",
+            "url": "https://twitter.com/4Jnet"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/fourjnetchannel"
+        }
+    ],
+    "tags": [
+        "defi"
```
Comment (bronejoost) [inline blockchains/smartchain/assets/0xbfb1a68962Fb4ED040FD3a0a71dC2C2015BCc667/info.json:22]: "metaverse",
		"nft"

**Comment by cryptomanz** [comment]
Please follow the rules for adding token, your logo should be a full screen, 256x256.

**Pull request closed without merging**